helm/storage.rst

```diff
@@ -48,7 +48,7 @@ planes:
 
   numDevices: The number of logical storage devices.  An advanced option.  This allows data to be striped over multiple storage locations.  If it is set, then it assumes pvc defines a set of pvcs: ``pvc-1`` .. ``pvc-<n>``, which are mounted at locations ``prefix/d1`` .. ``prefix/d<n>``.  (Not yet implemented and likely to change!)
 
-  labels:  A list of uses which the plane is used for.  On a simple system it may make sense to place dali, dlls and sasha all on the same volume.
+  category:  The uses which the plane is used for.  On a simple system it may make sense to place dali, dlls and sasha all on the same volume - in which case multiple planes can be defined with exactly the same definition, but different categories.
```

Comment:
The use which the plane is used for (which sounds a bit clumsy too, and could be worded better - perhaps "What the plane is used for")